### PR TITLE
Added an API route to get recent youtube videos by channel ID

### PR DESCRIPTION
**Pull request by jackwhelan**
Initial change was as described in title.
Also added Cron to this PR: [b833a83](https://github.com/RuadhanMulcahy/Chess-Video-Indexer/pull/18/commits/b833a83d0df775c34fa1e39a784b98c6ee767afe)

### CODE REVIEW

File: services/monitor/cron.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            self.sched.add_job(func, job_type, day=frequency)
+        else:
+            raise Exception(f'Cadence Type {cadence_type} is not valid.')
+
+    def stop_scheduler(self):
+        """
+        Cancel the job stored in this object.
+        """
+        self.sched.shutdown()
+
```
Comment (RuadhanMulcahy) [inline services/monitor/cron.py:43]: Is it necessary to have this method? Could this be simplified by just calling sched.shutdown directly from outside the class? ie. Cron(obj).sched.shutdown()

File: services/monitor/cron.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    def start_scheduler(self):
+        """
+        Start the job loop for scheduled jobs.
+        """
+        self.sched.start()
```
Comment (RuadhanMulcahy) [inline services/monitor/cron.py:48]: Is it necessary to have this method? Could this be simplified by just calling sched.scheduler directly from outside the class? ie. Cron(obj).sched.scheduler()

File: services/monitor/cron.py (hunk 1 above)
Comment (jackwhelan) [reply services/monitor/cron.py:43]: Yes, good shout. These can be removed.

Comment (jackwhelan) [reply services/monitor/cron.py]: Done

File: services/monitor/cron.py (hunk 2 above)
Comment (jackwhelan) [reply services/monitor/cron.py]: Done

**Review by RuadhanMulcahy — APPROVED** [review]

**Pull request merged**